client.go

```diff
@@ -0,0 +1,277 @@
+package globalsign
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"sync"
+	"time"
+)
+
+// Client GlobalSign sdk client.
+type Client struct {
+	sync.RWMutex
+	// httpClient used to communicate with the API.
+	httpClient *http.Client
+
+	// BaseURL base url of API.
+	BaseURL *url.URL
+
+	// userAgent user agent for client header request.
+	userAgent string
+
+	// authToken token for authorization.
+	authToken *string
+
+	// authTokenTs authentication token timestamp.
+	authTokenTs time.Time
+
+	// DSSService Digital Signing Service (DSS).
+	DSSService DSSService
+
+	// vault store identity information.
+	vault *IdentityVault
+
+	// Options.
+	options *ClientOptions
+}
+
+// ClientOptions options for client.
+type ClientOptions struct {
+	// BaseURL base url of API.
+	BaseURL *url.URL
+
+	// ApiKey API key credentials.
+	ApiKey string
+
+	// ApiSecret API secret credentials.
+	ApiSecret string
+
+	// CertFilePath path file to mTLS cert file.
+	CertFilePath string
+
+	// KeyFilePath path file to mTLS cert private key.
+	KeyFilePath string
+}
+
+// NewClient initiate client with `API Key`, `API Secret`, `Certificate file path`, `Private Key file path` and return globalsign sdk client.
```

Comment:
Should be "initiates" and then "returns the globalsign client"

Reply:
Updated at a3ddc191292d0952b56e2269c3cbfa6f517b98fa